### DISCUSSION TITLE
Need URLs used by Homebrew (company blocks outbound internet on servers; need to add URLs to whitelist)

### DISCUSSION THREAD

**Discussion by bripakes** [Homebrew/discussions#5498] [Getting started]
### Output of `brew config`

```shell
none
```

### Output of `brew doctor`

```shell
none
```

### Description of issue

My company blocks outbound internet on servers. I have a Mac Mini that has an AMP (Apache, MySQL, PHP) setup and I want to move to using Homebrew to install PHP and Apache HTTPD. I have Homebrew installed on the Mac Mini, but I can't run "brew update" since the URLs that Homebrew is calling are blocked. Would someone provide the URLs needed by Homebrew when running "brew update" and "brew install php apache"?  (so far I am seeing https://formulae.brew.sh and https://ghcr.io)  Thank you!

**Comment by bripakes** [comment]
Adding output for "brew doctor" (sorry it was not added above):
==> Downloading https://formulae.brew.sh/api/formula.jws.json

curl: (60) SSL certificate problem: self signed certificate in certificate chain
More details here: https://curl.haxx.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
Error: Failure while executing; `/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --user-agent Homebrew/4.3.9\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 10.15.7\)\ curl/7.64.1 --header Accept-Language:\ en --fail --progress-bar --remote-time --output /Users/administrator/Library/Caches/Homebrew/api/formula.jws.json --location --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.3.9\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 10.15.7\)\ curl/7.64.1 --header Accept-Language:\ en --fail --progress-bar --compressed --speed-limit 100 --speed-time 5 https://formulae.brew.sh/api/formula.jws.json` exited with 60. Here's the output:

curl: (60) SSL certificate problem: self signed certificate in certificate chain
More details here: https://curl.haxx.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.

**Comment by p-linnane** [comment]
`formulae.brew.sh` and `ghcr.io` will get you most of the way with a standard install. This will allow you run `brew update` and upgrade any formulae that we have bottles for.

**Comment by alan-czajkowski** [comment]
can someone please post a comprehensive list of hostnames?

**Reply by SMillerDev** [reply]
No such list exists within the open source project, and it seems clear here that nobody is volunteering the effort to create one.
